MVP Module/Source/Feed Presentation/FeedLoadErrorViewModel.swift

```
@@ -0,0 +1,5 @@
+//
+
+struct FeedLoadErrorViewModel {
+	let message: String
```

The `message ` could be an optional `String?`, so the Presenter can decide when to show/hide the message.

For example, in the `didStartLoadingFeed()` method, the Presenter can call  `loadErrorView.display(FeedLoadErrorViewModel(message: .none))` to hide the error.

This way, the ViewController doesn't need to decide when to hide the error. It's easier to manage state when all changes are centralized in a single component.